packages/code-du-travail-frontend/src/api/modules/enterprises/service/fetchEnterprises.ts

```diff
@@ -28,52 +27,47 @@ export const fetchEnterprises = async (
     postCode.length > 0 ? `&code_postal=${postCode.join(",")}` : ""
   }`;
 
-  try {
-    const fetchReq = await fetch(url);
-
-    const jsonResponse: ApiRechercheEntrepriseResponse = await fetchReq.json();
-
-    const entreprises = jsonResponse.results.map((result) => {
-      const conventions =
-        result.complements.liste_idcc?.map((idccNumber) => {
-          return {
-            idcc: parseInt(idccNumber, 10),
-            shortTitle: `Convention collective ${idccNumber}`,
-            id: idccNumber,
-            title: `Convention collective ${idccNumber}`,
-          };
-        }) ?? [];
+  const fetchReq = await fetch(url);
+  if (!fetchReq.ok) {
+    console.log("OK")
+    throw new Error(fetchReq.statusText);
```

Comment:
ça pourrait être pas mal d'avoir un peu plus de contexte dans le message d'erreur 

```suggestion
    throw new Error(`Erreur lors de la récupération des entreprises depuis annuaire-entreprise, code : ${fetchReq.status} (${fetchReq.statusText})`);
```